com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -770,6 +775,9 @@ protected override void Dispose(bool disposing)
 
             CleanupLightLoop();
 
+            ReleaseVolumetricClouds();
+            CleanupSubsurfaceScattering();
```

Comment:
can you do backport PR for 2021.1 and 2020.2 only for this CleanupSubsurfaceScattering();
 call here? (no need for changelog ,thanks)